packages/forms-web-app/src/pages/projects/section-51/index/view.njk

```diff
@@ -10,7 +10,8 @@
 {% from "components/wrappers/related-guides-wrapper.njk" import relatedGuidesWrapper %}
 {% from "projects/section-51/index/_components/results.njk" import sectionResults %}
 
-{% set pageTitle = title %}
+{% set pageTitle = t('section51Index.heading1') %}
+{% set noAdviceExists = adviceExists == false %}
```

Comment:
Sorry, missed that last time, you don't need to set `noAdviceExists`.
If you want to check for advice _not_ existing in a conditional statement, you can negate it in nunjucks like you would in javascript, but using different syntax:

`{% if(not adviceExists) %}
....
{% endif %}`